src/components/ui/Price/Price.stories.mdx

```diff
@@ -0,0 +1,185 @@
+    },
+    variant: {
+      description: 'The current variant for prices.',
+      options: ['installment', 'listing', 'savings', 'selling', 'spot'],
+      control: { type: 'select' },
+    },
+    formatter: {
+      table: { disable: true },
+    },
+  }}
+/>
+
+export const Template = (args) => <Price {...args} />
+
+<header>
+
+# Price
+
+Prices are used to display the price of a given product, discount and total values.
+
+</header>
+
+## Overview
+
+The `Price` component uses [FastStore UI Price](https://www.faststore.dev/reference/ui/atoms/Price) as base.
+
+<Canvas>
+  <Story name="overview-default">{Template.bind({})}</Story>
+  <Story
+    name="overview-installment"
+    args={{
+      variant: 'installment',
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+  <Story
+    name="overview-listing"
+    args={{
+      variant: 'listing',
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+  <Story
+    name="overview-savings"
+    args={{
+      variant: 'savings',
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+  <Story
+    name="overview-selling"
+    args={{
+      variant: 'selling',
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+  <Story
+    name="overview-spot"
+    args={{
+      variant: 'spot',
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+---
+
+## Usage
+
+`import Price from '../components/ui/Price'`
+
+<Canvas>
+  <Story
+    name="default"
+    args={{
+      variant: '',
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+<ArgsTable story="default" />
+
+---
+
+## Variations
+
+### Installment
+
+<Canvas>
+  <Story
+    name="installment"
+    args={{
+      variant: 'installment',
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+### Listing
+
+<Canvas>
+  <Story
+    name="listing"
+    args={{
+      variant: 'listing',
+      SRText: 'Original price:',
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+| Local token                                     | Default value/Global token linked                                                                        |
+| ----------------------------------------------- | -------------------------------------------------------------------------------------------------------- |
+| <code>--fs-price-listing-color</code>           | <div style={{ backgroundColor: 'var(--fs-color-text-light)' }} /><code>var(--fs-color-text-light)</code> |
+| <code>--fs-price-listing-text-decoration</code> | <code>line-through</code>                                                                                |
+
+### Savings
+
+<Canvas>
+  <Story
+    name="savings"
+    args={{
+      variant: 'savings',
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+### Selling
+
+<Canvas>
+  <Story
+    name="selling"
+    args={{
+      variant: 'selling',
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+### Spot
+
+<Canvas>
+  <Story
+    name="spot"
+    args={{
+      variant: 'spot',
```

Comment:
```suggestion
      variant: 'spot',
      SRText: 'Sale price:',
```

Actually, I'm not sure 🤔
The main reason why we have this `SRText` here is to help to identify what "type" of price is it when we have more than one price displayed. Maybe we could improve the description for this prop, lemme think.

Ex:
![image](https://user-images.githubusercontent.com/3356699/171485443-af96e0ae-47ad-4859-8ab8-8e64f0a869e6.png)

Reply:
Agree with you!